tuplex/codegen/src/LLVMEnvironment.cc

```diff
@@ -357,6 +361,131 @@ namespace tuplex {
             return retType;
         }
 
+        llvm::Type *LLVMEnvironment::getIteratorType(IteratorInfo *iteratorInfo) {
```

Comment:
is this lazy? I.e. createOrGetIteratorType?

Reply:
Refactored.